packages/nimble-blazor/Tests/NimbleBlazor.Tests/Unit/Components/NimbleAnchorTests.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Linq.Expressions;
+using Bunit;
+using Xunit;
+
+namespace NimbleBlazor.Tests.Unit.Components;
+
+/// <summary>
+/// Tests for <see cref="NimbleAnchor"/>
+/// </summary>
+public class NimbleAnchorTests
+{
+    [Fact]
+    public void NimbleAnchor_Rendered_HasAnchorMarkup()
+    {
+        var context = new TestContext();
+        context.JSInterop.Mode = JSRuntimeMode.Loose;
+        var expectedMarkup = "nimble-anchor";
+
+        var anchor = context.RenderComponent<NimbleAnchor>();
+
+        Assert.Contains(expectedMarkup, anchor.Markup);
+    }
+
+    [Theory]
+    [InlineData(AnchorAppearance.Default, "<nimble-anchor>")]
+    [InlineData(AnchorAppearance.Prominent, "appearance=\"prominent\"")]
+    public void AnchorAppearance_AttributeIsSet(AnchorAppearance value, string expectedAttribute)
```

Comment:
Maybe rename `expectedAttribute` to `expectedMarkup` as the first string contains no attributes?

Reply:
Done